### ISSUE TITLE
WinAppSDK 1.1 Preview 1 and Preview 2 breaks in-app Acrylic

### ISSUE THREAD

**Issue by michalleptuch** [microsoft/WindowsAppSDK#2412]
### Describe the bug

I upgraded WindowsAppSDK in my project to preview version of 1.1 and now I see XAML elements through AcrylicBrush. Button behind StackPanel is not clickable.

``` c#
<Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
  <Button
    x:Name="myButton"
    Background="Red"
    Click="myButton_Click"
    Content="Click me" />
  <StackPanel HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
    <StackPanel.Background>
      <AcrylicBrush TintColor="#202020" TintOpacity="0.5" />
    </StackPanel.Background>
  </StackPanel>
</Grid>
```

### Steps to reproduce the bug

1. In csproj file change version of WinAppSDK from 1.0.x to 1.1.0-preview1 or 1.1.0-preview2
2. Clean project

### Expected behavior

AcrylicBrush works as expected

### Screenshots

1.0.2
![Screenshot 2022-04-18 102636](https://user-images.githubusercontent.com/43097852/163782038-171854e1-b1ac-423a-ad91-cdb0dd196dfc.png)

1.1.0-preview1, 1.1.0-preview2
![Screenshot 2022-04-18 102805](https://user-images.githubusercontent.com/43097852/163782072-72c1a7f3-008d-4536-9a82-72a43231ec07.png)

### NuGet package version

1.1.0-preview1, 1.1.0-preview2

### Packaging type

Packaged (MSIX)

### Windows version

Windows 11 version 21H2 (22000)

### IDE

Visual Studio 2022 (the latest available udpate)

**Comment by codendone** [comment]
[[internal bug]](http://task.ms/39329255)

**Comment by codendone** [comment]
This is fixed in 1.1-stable.
